backend/spacewalk-backend.spec

```diff
@@ -93,8 +93,12 @@ Requires:       python3-rhnlib >= 2.5.74
 Requires:       python3-rpm
 %else
 Requires:       python2-rhnlib >= 2.5.74
-Requires:       %{name}-libs >= %{version}
+Requires:       %{name}-libs >= %{version
+%if 0%{?sle_version} >= 150000
```

Comment:
We usually make this check as:
```
%if 0%{?suse_version} >= 1320
```

Reply:
Mmmmm, I was having a look at https://en.opensuse.org/openSUSE:Packaging_for_Leap

Maybe we could use `%if 0%{?suse_version} >= 1500` to make it more clear?

Reply:
Fine with that :+1:

Reply:
As discussed at IRC, I just adapted all other >= 1320 with >= 1500